.github/workflows/swift-toolchain.yml

```diff
@@ -266,6 +266,10 @@ env:
   WORKAROUND_WINDOWS_PINNED_BOOTSTRAP_TOOLCHAIN_REPO: thebrowsercompany/swift-build
   WORKAROUND_WINDOWS_PINNED_BOOTSTRAP_TOOLCHAIN_RELEASE: 20231016.5
 
+  # Workaround for ... on Windows preventing us from using the 5.10 toolchain release.
```

Comment:
I believe @compnerd left the `...` to be filled with an actual reason.

Reply:
Done